### ISSUE TITLE
Windows: Make VSS-Provider (Volume Shadow Copy Service) configurable

### ISSUE THREAD

**Issue by meichenberger** [stepping-stone/sepiola#33]
If a QEMU-Guest-Agent is installed on a Windows 7 Machine (64-bit), the current Sepiola-Build 2.3.1 tries to use the QEMU-Guest-Agent VSS-Provider (Volume Shadow Copy Service) instead of the Microsoft VSS-Provider. This leads to the following error:

```
Volume Shadow Copy Service error: Error creating the Shadow Copy Provider COM class with
CLSID {...} [0x8007007e, The specified module could not be found.].
```

Due to the error, no backup can be executed.

As a short term workaround, please make the usage of the Volume Shadow Copy Service configurable:
- The new default is to have the VSS-Provider (Volume Shadow Copy Service) deactivated.
- The user gets the possibility to turn the VSS-Provider (Volume Shadow Copy Service) on (via a checkbox).

**Comment by dev-zero** [comment]
Since we will also have a snapshot feature on Linux at some point (either via LVM, BTRFS, ZFS or Snapper as an intermediary) we need this configuration interface to be somewhat generic from the start.
Furthermore it may be necessary to enable/disable VSS (or other snapshot technology) on a per-volume basis.

**Comment by dev-zero** [comment]
since we already shipped a version of Sepiola where snapshotting on Windows was mandatory, I strongly recommend to make it ENABLED by default on Windows and DISABLED (and GREYED OUT) on Linux and macOS

**Comment by dev-zero** [comment]
confirmed to be fixed in 2.5.0.rc0
